firefox-ios/Client/Frontend/Autofill/Address/Edit/EditAddressViewController.swift

```diff
@@ -48,27 +34,28 @@ class EditAddressViewController: UIViewController, WKNavigationDelegate, WKScrip
 
     override func viewDidLoad() {
         super.viewDidLoad()
+        listenForThemeChange(view)
+    }
 
+     override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
         setupWebView()
```

Comment:
Is there a reason we add the webview in viewWillAppear? Be careful that this method might be called multiple times and that can cause the views to be re-added.

Reply:
viewDidLoad() is called only once after the view loads, so it's a great candidate

Reply:
Updated in https://github.com/mozilla-mobile/firefox-ios/pull/20738/commits/9f66c577a2e51249cd0a6fafc44f0075e9e0a334